### PR TITLE
Implement Navbar to Plannerpage

### PR DESCRIPTION
**Pull request by lkk95**
In this PR I implemented a navbar for navigating to the plannerpage.
Homepage is empty at the moment, this will be tackled in another user story.

### CODE REVIEW

File: src/components/Layout/Header.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import logo from '../../imgs/logo.svg';
+
+import StyledHeader from './styledheader.js';
+
+export default function Header() {
+	return (
+		<StyledHeader>
+			<img src={logo} alt="PlatePlan logo with plate and cutlery" />
```
Comment (macfrei) [inline src/components/Layout/Header.js:8]: I would recommend to just write the text that is on the logo and not describe the logo. That is a bit overkill :)
```suggestion
			<img src={logo} alt="PlatePlan" />
```

File: src/components/Layout/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+export default function Navbar() {
+	return (
+		<StyledNav>
+			<StyledNavLink to="/">
+				<i className="fa-solid fa-house"></i>
+			</StyledNavLink>
+			<StyledNavLink to="/planner">
+				<i className="fa-solid fa-calendar-days"></i>
+			</StyledNavLink>
```
Comment (macfrei) [inline src/components/Layout/Navbar.js:12]: Maybe add some aria-labels here for better accessibility:
```suggestion
			<StyledNavLink to="/" aria-label="Home">
				<i className="fa-solid fa-house"></i>
			</StyledNavLink>
			<StyledNavLink to="/planner" aria-label="Planner">
				<i className="fa-solid fa-calendar-days"></i>
			</StyledNavLink>
```

**Review by macfrei — COMMENTED** [review]
Good job 👍

File: src/components/Layout/Navbar.js (hunk 2 above)
Comment (lkk95) [reply src/components/Layout/Navbar.js]: Hi Lene,
thanks for your review!
I forgot the accessibility point regarding the navbar, I implemented this and the other things now.

**Review by macfrei — APPROVED** [review]

**Pull request merged**